### DISCUSSION TITLE
i cant create data on lunar - Please install the suggested Algolia client: algolia/algoliasearch-client-php.

### DISCUSSION THREAD

**Discussion by qhoirulanwar** [lunarphp/lunar#1739] [Q&A] [answered]
i want to create some product or brand but i have error message
Please install the suggested Algolia client: algolia/algoliasearch-client-php.

![image](https://github.com/lunarphp/lunar/assets/22901097/11166559-49b5-4538-8704-4ae86f4229a8)

**Comment by alecritson** [comment]
Looks like your Scout is configured for Algolia but it's not set up. You can choose a Scout driver which works for you https://laravel.com/docs/11.x/scout

**Reply by qhoirulanwar** [reply]
Can I use lunar directly without connecting to scout or another full-text search?

**Reply by pippos44** [reply, marked as answer]
Yes, just use the "collection" while in local development or "database" Scout driver if you app is small https://laravel.com/docs/11.x/scout#collection-engine
